stock_manager/scripts/api/google_sheets_api.py

```diff
@@ -16,30 +15,26 @@ def main():
     # update_sheet(sheet)
 
 
-# read_sheet(sheet)
-
-
 def update_sheet(sheet: Worksheet):
     # sheet.update_cell(1, 1, 'Another Updated Value')
     sheet.update(range_name='A2:D2', values=[[1, 2, 3, 4]])
-
-
-# sheet.update(range_name='B1:C2', values=[['New Value 1', 'New Value 2'], ['New Value 3', 'New Value 4']])
-# sheet.batch_update([{
-# 	'range': 'D1',
-# 	'values': [['Batch Updated Value']]
-# }, {
-# 	'range': 'E1:F2',
-# 	'values': [['Batch', 'Update'], ['Example', 'Data']]
-# }])
+    
+    # sheet.update(range_name='B1:C2', values=[['New Value 1', 'New Value 2'], ['New Value 3', 'New Value 4']])
```

Comment:
Same comment as before: what's up with the commented out code that matters enough to keep correct but not enough to have a real codepath to? Is this really something you want to uncomment only in special situations instead of having an option to run it or not?